File: wallet_exchange/wallet.py
def main():
    t = int(input())
    for _ in range(t):
        a, b = map(int, input().split())
        if (a + b) % 2:
            print("Alice")
        else:
            print("Bob")
        
    
if __name__ == "__main__":
    main()

File: wallet_exchange/wallet.c
#include <stdio.h>

int main(void)
{
    int t;
    scanf("%d", &t);
    for (int i = 0; i < t; i++)
    {
        long long a, b;
        scanf("%lld%lld", &a, &b);
        if ((a + b) % 2)
        {
            printf("Alice\n");
        }

        else
        {
            printf("Bob\n");
        }
    }
}
